### ISSUE TITLE
Unable to get location when installing from testing repo

### ISSUE THREAD

**Issue by cassidyjames** [danrabbit/nimbus#20]
I'm getting this with the version in the testing repo:
```
(com.github.danrabbit.nimbus:3116): GWeather-WARNING **: Failed to get IWIN forecast data: 403 Forbidden
```

**Comment by cassidyjames** [comment]
Full output:

```
$ com.github.danrabbit.nimbus 

(com.github.danrabbit.nimbus:3161): Gtk-WARNING **: Theme parsing error: gtk-widgets.css:2045:14: 'min-height' is not a valid property name

(com.github.danrabbit.nimbus:3161): Gtk-WARNING **: Theme parsing error: gtk-widgets.css:2046:13: 'min-width' is not a valid property name

(com.github.danrabbit.nimbus:3161): Gtk-WARNING **: Theme parsing error: gtk-widgets.css:2051:13: 'min-width' is not a valid property name

(com.github.danrabbit.nimbus:3161): Gtk-WARNING **: Theme parsing error: gtk-widgets.css:2261:14: 'min-height' is not a valid property name

(com.github.danrabbit.nimbus:3161): Gtk-WARNING **: Theme parsing error: gtk-widgets.css:2262:13: 'min-width' is not a valid property name
Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.

** (com.github.danrabbit.nimbus:3161): WARNING **: MainWindow.vala:166: Failed to connect to GeoClue2 service: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.GeoClue2 was not provided by any .service files

(com.github.danrabbit.nimbus:3161): GWeather-WARNING **: Failed to get IWIN forecast data: 403 Forbidden
```

**Comment by cassidyjames** [comment]
Looks like you need `geoclue-2.0` explicitly as a dependency; after installing that it works. https://github.com/jonls/redshift/issues/318#issuecomment-262103785

**Comment by danrabbit** [comment]
Fixed in 39b06410a45327f788644e965162c0b9fae0483f
